### DISCUSSION TITLE
How to record multiple keydown events in Svelte?

### DISCUSSION THREAD

**Discussion by elite0421** [community/community#58883] [Programming Help] [answered]
### Body

I want to check if control and backspace both are pressed at the same time as well as check if the backspace is pressed once. Both the cases will call different function.

```
on:keydown={(e) => {
        if (e.key === 'Control') {
            if(e.key ==='Backspace')
            ctrl_backspace();
        }
        else if (e.key === 'Backspace') {
            backspace();
        }
    }
}

```
### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by iamsuiux** [comment, marked as answer]
Modifiers are included as separate properties of the event:

```
if (e.ctrlKey && !e.altKey && !e.shiftKey && e.key == 'Backspace') {
  // ctrl + backspace
}
if (!e.ctrlKey && !e.altKey && !e.shiftKey && e.key == 'Backspace') {
  // backspace
}
```

Would recommend also checking that the other modifiers are false for a precise match, the user might have tried to invoke something else otherwise.
